sdk/ios/Configsum/Configsum/ConfigsumTests/ConfigsumTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ConfigsumTests.swift
+//  ConfigsumTests
+//
+//  Created by Alexandru Savu on 2017-10-04.
+//  Copyright © 2017 Lifesum. All rights reserved.
+//
+
+import XCTest
+@testable import Configsum
+
+class ConfigsumTests: XCTestCase {
+    var configsum: Configsum!
+    
+    override func setUp() {
+        super.setUp()
+        let environment = Environment.test
+        self.configsum = Configsum(environment: environment, token: "")
+    }
+    
+    func testGetDefaultStringValue() {
+        do {
+            let stringValue = try self.configsum.getString(key: "stringVal1",
```

Comment:
Shouldn't these tests not only test the type of the return value but also the value itself? Or even the value itself would be sufficient.

Reply:
these functions will always return something. your are correct in saying that we should also test for matching return with default value. but these tests here just test type correctness

Reply:
Yes, precisely. But the name of the test itself implies that it checks the value, not the type.

Reply:
always with keen eye. thanks